packages/meditrak-server/src/routes/dashboardRelations/assertDashboardRelationsPermissions.js

```diff
@@ -72,7 +72,7 @@ export const assertDashboardRelationGetPermissions = async (
   }
 
   throw new Error(
-    `Requires any of the permission groups (${dashboardRelation.permission_groups()}) access to the dashboard root entity code '${
+    `Requires any of the permission groups (${dashboardRelation.permission_groups.toString()}) access to the dashboard root entity code '${
```

Comment:
Oh so that didn't work @chris-bes ?

Reply:
Yeah it didn't 😬 . My bad for not testing it

Reply:
Perhaps it needed `dashboardRelation.permission_groups` (remove the parentheses)? If `a` is an array, `${a}` and `a.toString()` should be the same

Anyway, let's leave this as is 😃